test/evaluation/markForSubmissionEvaluationHandler.spec.ts

```diff
@@ -1,11 +1,11 @@
 import fs from 'fs';
 
-import { PresentationDefinition } from "@sphereon/pe-models";
+import { PresentationDefinition } from '@sphereon/pe-models';
```

Comment:
this file contains lots of noise

Reply:
I think these are fixes automatically applied by yarn fix.

Reply:
yes, but you can choose what to commit